scripts/test/mender-convert-qemu

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$DISK_IMG" ]; then
+    echo "ERROR: DISK_IMG is not set"
+    exit 1
+fi
+
+# Try common locations for OVMF
+ovmf_file=""
+for maybe_ovmf_file in /usr/share/OVMF/OVMF_CODE.fd /usr/share/ovmf/x64/OVMF_CODE.fd; do
+    if [ -f $maybe_ovmf_file ]; then
+        ovmf_file=$maybe_ovmf_file
+    fi
+done
+
+if [ -z "${ovmf_file}" ]; then
+    echo "ERROR: could not find OVMF file"
+    exit 1
+fi
+
+qemu-system-x86_64 \
+    -enable-kvm \
+    -nographic \
+    -m 256 \
+    -net user,hostfwd=tcp::8822-:22 \
+    -net nic,macaddr=52:54:00$(od -txC -An -N3 /dev/urandom|tr \  :) \
+    -bios ${ovmf_file} \
+    -drive format=raw,file=${DISK_IMG} &
+
+qemu_pid=$!
+
+trap qemu_cleanup HUP KILL TERM
+
+qemu_cleanup() {
+    kill -s TERM $qemu_pid
```

Comment:
technically it is possible for qemu to hang or otherwise not respond to TERM, I would sleep here for 2 or 4 seconds and then `kill -0 $qemu_pid 2>/dev/null && kill -9 $qemu_pid; sleep 2; kill -0 $qemu_pid 2>/dev/null && echo "qemu pid $qemu_pid still present";`

Reply:
This only happens once (at the end of all the tests) and if it does not respond the test will finish anyway and the VM be teared down, right? So I don't think it is that important, and I'd rather keep it simple.